pysteps/nowcasts/extrapolation.py

```diff
@@ -79,7 +81,7 @@ def forecast(precip, velocity, num_timesteps,
 
     if measure_time:
         computation_time = time.time() - start_time
-        print(f"{computation_time:.2f} seconds.")
+        _LOGGER.debug("%.2f seconds.", computation_time)
```

Comment:
Or we could remove the measure_time argument and log the computation time at DEBUG level, what do you think?

Reply:
That's a good point. But we should leave the keyword argument and add a deprecation warning to maintain backward compatibility (at least for a few releases).

Reply:
Agreed ;-)

Reply:
So a `warnings.warn` with `PendingDeprecationWarning` for one versione, `DeprecationWarning` for the next versione and total remove with the third version?
Or is best to use a `DeprecationWarning` for the next two version and then remove?

Reply:
Good point. According to [PEP-565](https://www.python.org/dev/peps/pep-0565/):
- `PendingDeprecationWarning` is hidden by default for all code.
- `DeprecationWarning` is reported by default for code that runs directly in the `__main__` module (as such code is considered relatively unlikely to have a dedicated test suite), but hidden by default for code in other modules.
- `FutureWarning` is reported by default for all code. The intended audience is users of applications written in Python, rather than other Python developers 

So, since we want to target the users, it seems that [FutureWarning](https://www.python.org/dev/peps/pep-0565/#additional-use-case-for-futurewarning) is the way to go.